.ci/openshift_integration.sh

```diff
@@ -69,4 +69,4 @@ $(realpath odo) registry delete DefaultDevfileRegistry -f
 $(realpath odo) registry add TestDevfileRegistry http://$REGISTRY_HOSTNAME
 
 # Run the devfile validation tests
-ENV=openshift REGISTRY=remote tests/check_odov2.sh $(realpath odo) $YQ_PATH
+ENV=openshift REGISTRY=remote ENABLE_TLS="false" tests/check_odov2.sh $(realpath odo) $YQ_PATH
```

Comment:
I took a second look here. @michael-valdron out of curiosity, could we combine the two issues in the same PR (https://github.com/devfile/api/issues/1664 is the other one). Meaning we could also migrate the check to odo3 here right? This way I guess the ci check will be unblocked

Reply:
@thepetk Yeah I can add the change to odov3 testing as well to unblock the testing entirely and resolve both. I'll work on and add it today.